test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -274,30 +271,37 @@ def convert_date_time(cls, dt=None, fmt=None, syslog=False):
         else:
             return int(tm)
 
-    def get_syslog_date_str(self, logline=None):
-        if logline is None:
-            return None
-
-        if "T" in logline:
-            return logline[:19]
-        else:
-            return logline[:15]
-
-    def convert_syslog_time(self, dt=None):
-        if dt is None:
-            return None
+    def convert_syslog_date_str(self, logline):
+        """
+        Convert syslog time string in logline to the  str time format
+        required by the convert_date_time(),
+        which will convert it to epoch time.
+        The syslog time can be in 2 formats-
+        1) RFC 5424- "yyyy-mm-ddThh:mm:ss-TZ"
+        2) "Month date hh:mm:ss" (with no year)
+        :param logline: the syslog line from which to read date
+        :type logline: str or None
+        :returns: str
+        """
 
-        if 'T' in dt:
+        if "T" in logline.split(' ', 1)[0]:
+            # if time format is RFC 5424 -
+            # remove unwanted strings 'T' and time zones from dt
+            # before converting it to datetime
+            dt = logline[:19]
             dt_str = dt[:10] + " " + dt[12:19]
-            t = datetime.datetime.strptime(dt_str, "%Y-%m-%d %H:%M:%S")
-            return t
+            t = datetime.strptime(dt_str, "%Y-%m-%d %H:%M:%S")
+            print("time format is: " + str(t))
+            return str(t)
         else:
-            t = datetime.datetime.strptime(dt, "%b %d %H:%M:%S")
+            dt = logline[:15]
+            t = datetime.strptime(dt, "%b %d %H:%M:%S")
+            # Since there is no year, strptime adds date as 1900.
```

Comment:
please print a warning here mentioning that you are assuming current year since year is missing, this will help people debug potential test failures when reading logs from last year